hedera-node/hedera-app/src/main/java/com/hedera/node/app/Hedera.java

```diff
@@ -305,8 +304,8 @@ private void onStateInitialized(
                     // We exited from this method early if we were recovering from an event stream.
                 case EVENT_STREAM_RECOVERY -> throw new RuntimeException("Should never be reached");
             }
-        } catch (final Throwable th) {
-            logger.fatal("Critical failure during initialization", th);
+        } catch (final Exception e) {
+            logger.fatal("Critical failure during initialization", e);
             System.exit(CRITICAL_FAILURE_EXIT_CODE);
```

Comment:
Yes, I think it makes sense to keep `Throwable` in places where we exit directly. This would happen with an uncaught `Error` anyway, and we get at least the log statement before this happens.

Reply:
Updated.